doc/man/8/ceph.rst

```diff
@@ -928,6 +928,16 @@ Usage::
 Subcommand ``pool`` is used for managing data pools. It uses some additional
 subcommands.
 
+Subcommand ``application enable`` enables the pool for one of the applicatons -  object, block or file storage.
+
+Usage::
+        ceph osd pool application enable {pool-name} {rgw|rbd|cephfs}
+
+Subcommand ``application disable`` disables the given application -- object, block or file storage -- on the given pool.
+
+Usage::
+        ceph osd pool application disable {pool-name} {pool-name} {rgw|rbd|cephfs}
```

Comment:
Still not correct: I've written {pool-name} twice. I don't have a working ceph cluster right now. Once I have it, I'll verify and update the PR.